lib/skate_web/router.ex

```diff
@@ -3,6 +3,10 @@ defmodule SkateWeb.Router do
   use Plug.ErrorHandler
   use Sentry.Plug
 
+  # We prefer to redirect at the load balancer level, but for security through
+  # redundancy, we're keeping this plug just in case.
+  # A note: this does not affect anything in `endpoint.ex`, meaning static assets
+  # do not get redirected to HTTPS with this config currently.
```

Comment:
**nitpick:** `endpoint.ex` is actually the place that invokes `plug SkateWeb.Router`, so I think it might be more accurate to say that `Plug.Static` specifically isn't affected.

Reply:
WWhat do you think about ba72a37e9968f37a6e9d9dbbe2b1f2663f51637a ?

Reply:
Looks good!